assets/js/graph.js

```diff
@@ -9,7 +9,8 @@ async function drawGraph(
   const container = document.getElementById('graph-container')
 
   const { index, links, content } = await fetchData
-  const curPage = window.location.href.replace(baseUrl, "").replace(/\/$/g, "")
+  // Use .pathname to remove hashes / searchParams / text fragments
+  const curPage = window.location.pathname.replace(/\/$/g, "")
```

Comment:
`pathname` ignores cases where Quartz is hosted on a path like `<username>.github.io/quartz`

Reply:
Should be fixed in [aaed5dc](https://github.com/jackyzha0/quartz/pull/124/commits/aaed5dc1f1849a54869743596a6133548e83392d) and [2b5c03c](https://github.com/jackyzha0/quartz/pull/124/commits/2b5c03c97286e06e94e8a27634678e64473a2ec8). `pathname` is now used to reconstruct the whole url, then trailing slash and base url are removed.

Reply:
Wow if this works, this is a much cleaner construction then passing in `{{.BaseURL}}` from Hugo to all client-side JS, wondering if this is worth doing at some point